apps/admin-gui-e2e/cypress.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig } from 'cypress';
+import { nxE2EPreset } from '@nrwl/cypress/plugins/cypress-preset';
+
+export default defineConfig({
+  e2e: {
+    ...nxE2EPreset(__dirname),
+    fileServerFolder: '.',
+    fixturesFolder: './src/fixtures',
+    video: true,
+    videosFolder: '../../dist/cypress/apps/admin-gui-e2e/videos',
+    screenshotsFolder: '../../dist/cypress/apps/admin-gui-e2e/screenshots',
+    chromeWebSecurity: false,
+    baseUrl: 'http://localhost:4200',
+    env: {
+      BA_USERNAME: 'perun',
+      BA_PASSWORD: 'test',
+      BA_USERNAME_VO_MANAGER: 'voManager',
+      BA_PASSWORD_VO_MANAGER: 'test',
+      BA_USERNAME_GROUP_MANAGER: 'groupManager',
+      BA_PASSWORD_GROUP_MANAGER: 'test',
+      BA_USERNAME_FACILITY_MANAGER: 'facilityManager',
+      BA_PASSWORD_FACILITY_MANAGER: 'test',
+      BA_USERNAME_RESOURCE_MANAGER: 'resourceManager',
+      BA_PASSWORD_RESOURCE_MANAGER: 'test',
+      BA_PASSWORD_TOP_GROUP_CREATOR: 'topGroupCreator',
```

Comment:
**Not a part of your PR, so ignore it.**
We should use USERNAME/PASSWORD for consistency, we should use it also in the test file, but for this, we need to change `perun.passwd` file in gitlab repo, so we can change it further.
```suggestion
      BA_USERNAME_TOP_GROUP_CREATOR: 'topGroupCreator',
      BA_PASSWORD_TOP_GROUP_CREATOR: 'test',
```

Reply:
I agree